scripts/deployment/web/forever.json

```diff
@@ -12,13 +12,53 @@
     "errFile": "/home/ec2-user/log/flashcrow/move-backend.err"
   },
   {
-    "id": "move-reporter",
+    "id": "move-reporter-0",
     "append": true,
     "watch": false,
     "command": "node -r ./globalWindowShim -r esm -r module-alias/register",
     "script": "reporter/reporter.js",
     "sourceDir": "/home/ec2-user/flashcrow",
-    "pidFile": "/home/ec2-user/log/flashcrow/move-reporter-forever.pid",
+    "args": ["--port", 8082],
+    "pidFile": "/home/ec2-user/log/flashcrow/move-reporter-0-forever.pid",
+    "logFile": "/home/ec2-user/log/flashcrow/move-reporter-forever.log",
```

Comment:
not sure yet whether these different `forever` tasks can share `logFile`, `outFile`, and `errFile`.

also, can we shorten some of these names / paths?